src/renderer/components/analytics/AnalyticsComponentRenderer.tsx

```
@@ -0,0 +1,853 @@
+              dataKey={yCol}
+              nameKey={xCol}
+              cx="50%"
+              cy="50%"
+              innerRadius={innerRadius}
+              outerRadius={120}
+              paddingAngle={2}
+              label
+            >
+              {data.map((_, i) => (
+                <Cell
+                  key={`cell-${i}`}
```

Biome (2.5.0)

[error] Avoid using the index of an array as key property in an element.

(lint/suspicious/noArrayIndexKey)